artifacts/CAMARA_common.yaml

```diff
@@ -2,12 +2,12 @@ openapi: 3.0.3
 info:
   title: Camara common data types
   description: Common data types for Camara APIs
-  contact:
-    email: sp-com@lists.camaraproject.org
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 0.4.0-wip
+  version: 0.5.0

```

Comment:
can we put the version and x-camara-commonalities sections together, and before the description (as this may be long) please ? I have the same basic comment on the example in the section 11.1 in the API-design-guidelines.md - will put that as a comment on the PR as well.

Reply:
will be handled in https://github.com/camaraproject/Commonalities/issues/264